libwallet/assets/wallet/asset_interface.go

```diff
@@ -22,6 +22,8 @@ type Asset interface {
 	SetSpecificPeer(address string)
 	GetExtendedPubKey(account int32) (string, error)
 	IsSyncShuttingDown() bool
+	EnableSyncShuttingDown()
+	EndSyncShuttingDown()
```

Comment:
EnableSyncShuttingDown and EndSyncShuttingDown are confusing to me, why not StartCancelSync and EndCancelSync